### PR TITLE
0.3.0

### PR DESCRIPTION
**Pull request by riceboyler**
Renaming pertinent component to AdvancedMaterialTable to attempt to clean up the types.

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -294,6 +294,6 @@ export interface Localization {
   };
 }
 
-export default class MaterialTable<RowData extends object> extends React.Component<
-  MaterialTableProps<RowData>
-> {}
+export default class AdvancedMaterialTable<
```
Comment (gunn4r) [inline types/index.d.ts]: In the usage code this is not a default export.

Comment (gunn4r) [reply types/index.d.ts]: ie:
```
import {
  AdvancedMaterialTable,
  AdvancedMaterialTableProps,
} from 'material-ui-advanced-table';
```

Comment (riceboyler) [reply types/index.d.ts]: Fixed.

File: README.md [hunk 2]
```diff
@@ -148,13 +148,13 @@ Here is a basic example of using material-ui-advanced-table within a react appli
 ```jsx
 import React, { Component } from "react";
 import ReactDOM from "react-dom";
-import MaterialTable from "material-ui-advanced-table";
+import AdvancedMaterialTable from "material-ui-advanced-table";
```
Comment (gunn4r) [inline README.md]: Same thing here... it's not a default export anymore it looks like.

Comment (gunn4r) [inline README.md]:
```suggestion
import { AdvancedMaterialTable } from "material-ui-advanced-table";
```

**Pull request merged**